ci/go-publisher-ossf-slsa3.yml

```diff
@@ -0,0 +1,22 @@
+name: SLSA Go releaser
```

Comment:
Since this workflow also publishes the built asset, please put a suffix of `-publish`. See [guidance](https://github.com/actions/starter-workflows/blob/main/.github/pull_request_template.md).

May be `go-ossf-slsa3-publish.yml`

Reply:
Done.